src/icp_canister_backend/tests/utils.rs

```diff
@@ -86,6 +86,39 @@ pub fn get_current_episode(pic: &PocketIc, canister_id: Principal, user: Princip
     candid::decode_one(&current_episode_result).unwrap()
 }
 
+pub fn get_stakable_episode(
+    pic: &PocketIc,
+    canister_id: Principal,
+    user: Principal,
+    relative_episode: u8,
+) -> u64 {
+    if relative_episode > 8 {
+        panic!("Relative episode must be 0-8");
+    }
+
+    let current_episode = get_current_episode(pic, canister_id, user);
+
+    let mut first_stakable = current_episode;
+    while first_stakable % 3 != 2 {
+        first_stakable += 1;
+    }
+
+    let absolute_episode = first_stakable + (relative_episode as u64 * 3);
+
+    absolute_episode
+}
+
+pub fn time_to_advance_for_episode(
+    pic: &PocketIc,
+    canister_id: Principal,
+    user: Principal,
+    target_episode: u64,
+) -> u64 {
+    let current_episode = get_current_episode(pic, canister_id, user);
+    let episodes_to_advance = target_episode - current_episode + 1;
```

Comment:
This function should return the exact time to advance to meet the episode end.
Right now it doesn't return meaningful value.